apps/locals/tests/test_dashboard_favorites.py

```
@@ -9,6 +9,62 @@
+    def setUp(self):
+        self.user = get_user_model().objects.create_superuser(
+            username="dashboardperf",
+            email="dashboardperf@example.com",
+            password="password",
+        )
+        self.client.force_login(self.user)
+
+    def test_dashboard_uses_precomputed_content_type_map(self):
+        get_for_models_original = ContentType.objects.get_for_models
+
+        def guarded_get_for_model(*args, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `args`

(ARG001)